docs/governance/how-is-etherlink-governed.md

```diff
@@ -68,11 +90,58 @@ The security governance process is like the kernel governance process, with thes
 - The quorum and supermajority requirements are higher
 - The timeframes are shorter
 
-These changes ensure expedited resolution of urgent issues while upholding integrity by demanding higher quorum to prevent potential nefarious actions.
+### Periods
+
+The security governance process has the same Proposal, Promotion, and Cooldown periods as the kernel governance process, but the lengths of these periods are different.
+The lengths are stored in the [security governance contract](https://better-call.dev/mainnet/KT1N5MHQW5fkqXkW9GPjRYfn5KwbuYrvsY1g).
+This table shows the period lengths as of the Bifröst Etherlink update and the Tezos Quebec protocol:
+
+Period | Length | Approximate time
+--- | --- | ---
+Proposal | 2560 layer 1 blocks | About 5.5 hours
+Promotion | 2560 layer 1 blocks | About 5.5 hours
+Cooldown | 86400 seconds | About 1 day
```

Comment:
TBF 86400 seconds is exactly 1 day. I don't know how the contract exactly uses the constant, but at face value `About 1 day` look strange

Reply:
There's a note after the prior table about why this might be longer than a day. Not sure whether to repeat that note here.